XenAdmin/TabPages/AdPage.resx

```diff
@@ -145,56 +145,61 @@
   <data name="ColumnExpand.HeaderText" xml:space="preserve">
     <value />
   </data>
-  <data name="ColumnExpand.MinimumWidth" type="System.Int32, mscorlib">
-    <value>20</value>
-  </data>
   <data name="ColumnExpand.Width" type="System.Int32, mscorlib">
-    <value>20</value>
+    <value>5</value>
   </data>
   <metadata name="ColumnTypeImage.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
     <value>True</value>
   </metadata>
+  <data name="dataGridViewCellStyle3.NullValue" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">

```

Comment:
Is this image needed and does it need to be embedded in the resx?

Reply:
I'm not sure, it was generated by the designer and I only left it because there were more like this in the file. I'll try to remove all of them and see what happens.